docs/en_us/platform_api/source/change_log.rst

```diff
@@ -12,6 +12,9 @@ Change Log
 
    * - Date
      - Change
+   * - 01 April 2016
+     - Marked the mobile API, the profile image API, and the course structure
+       API as deprecated.
```

Comment:
^ needed? we've stopped maintaining the change logs in the other guides (as of 12/31/16)